src/components/stack/stack.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { modesDarkDefault } from "../../../.storybook/utils";
+import readme from "./readme.md";
+import { html } from "../../../support/formatting";
+import { storyFilters } from "../../../.storybook/helpers";
+
+export default {
+  title: "Components/Stack",
```

Comment:
WDYT about adding this under `Components/Support/` to better convey it's purpose for now?